modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -280,6 +309,20 @@ export default {
       this.currentPage = 1;
 
       this.getPieces();
+    },
+    updateSlatList(items) {
```

Comment:
This isn't necessarily updating the slat list, as the name suggests. It's emitting an updated `items` array.

Reply:
You're right. In fact, it should be `updateSelectedItems`. And the current `updateSelectedItems` should be `updatedCheckbox` and should call the new `updateSelectedItems` to update `this.selectedItems` and emit the event. What do you think?

Reply:
This is getting confusing. We definitely need the refactor to keep things separate. I'll do that as my next project.

For now, please add a note before both `updateSelectedItems` and `updateSlatList` to move them into the new `AposRelationshipManager` in the refactor.

Reply:
Done